blockprotocol/blockprotocol
# Make Schema Table Responsive

## What this Does

Users inside blockprotocol can create or edit schemas for various entities. However, the current view of the schema pages breaks in mobile view. This PR is responsible for resolving the styling issues, and making the schema editor and viewing table responsive.

- [Make the text display responsive](https://github.com/blockprotocol/blockprotocol/pull/167/files#diff-309a59f94b4cdf913ac3ca9228a73fd75b2436c0ec3ed0a525abbaf39a3a8736R32)
- [Add overflow-x to parent](https://github.com/blockprotocol/blockprotocol/pull/167/files#diff-03a91238518a41200c3aadcccca251ca5002dd0da450e8b9ec637fb944b2572cR284)
- [Add min-width to input fields](https://github.com/blockprotocol/blockprotocol/pull/167/files#diff-3f0032dfe916053af5adc7019965db8e5014d77e8f93e3cde14fc3d1547ea0f5R907)

## How to Test this?

- Open up a user's schema or a different user's schema.
- Test responsiveness for various screen sizes.

![chrome_n08SH5F3ZB](https://user-images.githubusercontent.com/22196279/151924594-c94e6188-7644-4233-9244-7c07e7ceb9f3.gif)
![chrome_eu1LGw6Oxf](https://user-images.githubusercontent.com/22196279/151924634-0c537cdc-f785-4ec3-a20c-4b7377ae3f58.gif)



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201303965906637
